### PR TITLE
Added automatic CLA filtering option in Comm

### PR DESCRIPTION
**Pull request by kingofpayne**
Most of the applications check for the value of CLA byte of incoming APDUs and return an error status word when it is different from a constant.

This feature can help simplifying apps by removing the need to handle bad CLA errors, as there is not added value by the apps in dealing with this.

### CODE REVIEW

**Comment by agrojean-ledger** [comment]
@kingofpayne you need to rebase your branch

File: ledger_device_sdk/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ledger_device_sdk"
-version = "1.2.1"
+version = "1.3.0"
```
Comment (agrojean-ledger) [inline ledger_device_sdk/Cargo.toml:3]: You need to bump to `1.4.0` (it's already `1.3.0` on master)

File: ledger_device_sdk/src/io.rs [hunk 2]
```diff
@@ -129,15 +136,36 @@ pub struct ApduHeader {
...
+    /// * `cla` - Expected value for APDUs CLA byte.
+    ///
+    /// # Examples
+    ///
+    /// This method can be used when building an instance of [`Comm`]:
+    ///
+    /// ```
+    /// let mut comm = Comm::new().expect_cla(0xe0);
+    /// ```
+    pub fn expect_cla(mut self, cla: u8) -> Self {
```
Comment (agrojean-ledger) [inline ledger_device_sdk/src/io.rs:164]: I would rather call this function `set_expected_cla`

File: ledger_device_sdk/Cargo.toml (hunk 1 above)
Comment (kingofpayne) [reply ledger_device_sdk/Cargo.toml]: Done.

File: ledger_device_sdk/src/io.rs (hunk 2 above)
Comment (kingofpayne) [reply ledger_device_sdk/src/io.rs]: Good suggestion. Done.

**Review by agrojean-ledger — APPROVED** [review]

**Pull request merged**